src/database/entities/users.entity.ts

```diff
@@ -19,18 +19,21 @@ export class User {
   @Exclude()
   @Column({
     nullable: false,
```

Comment:
it could be true if register with google

Reply:
update